plenum/server/consensus/ordering_service.py

```diff
@@ -638,15 +638,16 @@ def signal_suspicious(req):
         elif why_not == PP_CHECK_NOT_NEXT:
             pp_view_no = pre_prepare.viewNo
             pp_seq_no = pre_prepare.ppSeqNo
-            last_pp_view_no, last_pp_seq_no = self.__last_pp_3pc
-            if pp_view_no >= last_pp_view_no and (
-                    self.is_master or self.last_ordered_3pc[1] != 0):
-                seq_frm = last_pp_seq_no + 1 if pp_view_no == last_pp_view_no else 1
+            _, last_pp_seq_no = self.__last_pp_3pc
+            # if pp_view_no >= last_pp_view_no and (
```

Comment:
Do we check somewhere that `pp_seq_no  > last_pp_seq_no `, that is this is not a PrePrepare with a ppSeqNo we've already received?

Reply:
we already did this check in _is_next_preprepare call.